server/pulp/server/webservices/views/users.py

```diff
@@ -17,6 +20,34 @@
 USER_WHITELIST = [u'login', u'name', u'roles']
 
 
+def serialize(user):
+    """
+    This function accepts a user object, adds a link to it, removes sensitive information from it,
+    and returns the modified object.
+
+    :param user: A user document
+    :type  user: bson.BSON
+    :return:     A modified version of the User, suitable for returning via the REST interface.
+    :rtype:      bson.BSON
+    """
+    import logging
```

Comment:
I fixed this.

It looks like I did something in a test that might not work on older versions of Django, so I'll see if I can get this working on EL 6 now.